app/bundles/CampaignBundle/Controller/EventController.php

```diff
@@ -120,7 +120,7 @@ public function newAction()
                 $formThemes[] = $event['settings']['formTheme'];
             }
 
-            $viewParams['form']             = $this->setFormTheme($form, 'MauticCampaignBundle:Campaign:index.html.php', $formThemes);
+            $viewParams['form']             = $this->setFormTheme($form, 'MauticCampaignBundle:Campaign:index.html.php', $formThemes); // @ticket TPROD-308
```

Comment:
Why wasn't this refactored to Twig? I don't get the comment either

Reply:
It's the oddest thing and having a little trouble tracking down the root cause. Looking at a few other bundles, it seems that the php template was used for this. Using a twig template (event a twig template with nothing it in) throws a 500. I'll push up what I have but could use help resolving.

Reproduce, when the Campaign Builder is open (either a new campaign or editing an existing campaign) when a new event is added/edited, it will fail to load the form.

Reply:
I also noticed this before you pushed the fix. The FormHelper has this deprecation notice in it:

```
'The '.FormHelper::class.' class is deprecated since version 4.3 and will be removed in 5.0; use Twig instead.'
```
I'm not quite sure what it means as I don't have much experience with handling Syfmony forms via Twig.

Reply:
I see a few more places that reference the php templates. The `DashboardListener` should get resolved view #11791 

Made a few other updates, I'll keep chipping away at this one.

Reply:
> I also noticed this before you pushed the fix. The FormHelper has this deprecation notice in it:
> 
> ```
> 'The '.FormHelper::class.' class is deprecated since version 4.3 and will be removed in 5.0; use Twig instead.'
> ```
> 
> I'm not quite sure what it means as I don't have much experience with handling Syfmony forms via Twig.

Right now the symfony version is 4.4. Once the templates are converted to twig, we would need to removed as many of the direct deprecation errors as we can. Some of the deprecated message are indirect and will go away once upgraded. For this one, I'm not sure without digging into the message to see where it comes from.